packages/components/pages/f-loyalty/src/tenantConfig.js

```diff
@@ -46,4 +46,30 @@ const bagValues = {
 
 const tenantBagFees = tenant => bagValues[tenant];
 
-export default tenantBagFees;
+const ordinals = {
+    'en-GB': true,
+    'en-AU': true,
+    'en-NZ': true,
+    'es-ES': false,
+    'en-IE': true,
+    'it-IT': false
+};
+
+const currencySymbolBefore = {
+    'en-GB': true,
+    'en-AU': true,
+    'en-NZ': true,
+    'es-ES': false,
+    'en-IE': true,
+    'it-IT': false
+};
+
+const tenantOrdinals = tenant => ordinals[tenant];
+
+const tenantCurrencySymbol = tenant => currencySymbolBefore[tenant];
```

Comment:
I think this should be named more explicitly - `tenantCurrencySymbol` sounds like it should return a single-character string rather than a boolean! I would suggest `tenantCurrencySymbolBefore` to persist the naming you've got internally.

Reply:
I would also say that adding "is" to the start would make things even clearer, i.e., `isCurrencySymbolBefore` and `isTenantCurrencySymbolBefore`

Reply:
I like the is also so will do both